scripts/bnsd/start.sh

```diff
@@ -3,7 +3,7 @@ set -o errexit -o nounset -o pipefail
 command -v shellcheck > /dev/null && shellcheck "$0"
 
 # Choose from https://hub.docker.com/r/iov1/tendermint/tags/
-export BNSD_TM_VERSION=v0.31.5
+export BNSD_TM_VERSION=v1.0.0
 # Choose from https://hub.docker.com/r/iov1/bnsd/tags/
 export BNSD_VERSION=v0.25.1
```

Comment:
`bnsd` version should be `export BNSD_VERSION=v1.0.0`

Reply:
You are right. Too confusing names.